.github/workflows/main.yml

```diff
@@ -35,19 +35,11 @@ jobs:
         shell: bash
         env:
           ACTUAL: ${{ steps.convert.outputs.text }}
-          EXPECTED: |
-            *Heading 1*
-
-            *Heading 2*
-
-            <http://example.com|Link>
-
-            •   List Item 1
-            •   List Item 2
+          EXPECTED: '*Heading 1*\n\n*Heading 2*\n\n<http://example.com|Link>\n\n•   List Item 1\n•   List Item 2\n'
```

Comment:
Thanks @denist-huma for your response. 

To give you more context the input that I'm getting is from Github cli Release notes Example : 

`{"body":"## What's Changed\n* Test slakify Text by @bachiri in [https://github.com/repo/pull/22\n\n\n**Full]..... `

So when applying the Slakify action into the input using the version `1.0.0` it works perfectly(The second screen shoot) but when I apply the second version `1.0.2` I got the result in the first screenshot . 

If I'm not mistaken the changes you have done are specific to your case because from what I understood the this action is just a wrapper of [slackify-markdown](https://github.com/jsarafajr/slackify-markdown) which is the best place to push your change(If it applies ).

Reply:
@bachiri GitHub/Gitlab are expecting to get markdown, WYSIWYG, a string code "\n" for example. But Slack API expects to receive a readable "\\n" in string code. Please look https://api.slack.com/reference/surfaces/formatting#line-breaks. 

This leads to that you got to process text that can be displayed in GitHub for Slack API to not to break.

My input from #8 has 2 line breaks in it.

```code
**Your server API is ready to use.✅**

The server API links:
```

Reply:
> If I'm not mistaken the changes you have done are specific to your case because from what I understood this action is just a wrapper of [slackify-markdown](https://github.com/jsarafajr/slackify-markdown) which is the best place to push your change(If it applies ).

I disagree that the changes I have done are specific to my case. I think that it should fix the bug for everybody who use multi-line strings.